magefiles/terraformplan.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	hversion "github.com/hashicorp/go-version" //nolint:gomodguard // hc-install uses hashicorp/go-version
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/aquasecurity/trivy/internal/testutil"
+)
+
+const (
+	terraformVersion       = "1.7.3"
+	terraformParallelLimit = 5
+
+	tfplanFile = "tfplan"
+)
+
+func fixtureTerraformPlanSnapshots(ctx context.Context) error {
+
+	if err := os.Setenv("TESTCONTAINERS_RYUK_DISABLED", "true"); err != nil {
+		return err
+	}
+
+	localstackC, addr, err := testutil.SetupLocalStack(ctx, "3.1.0")
+	if err != nil {
+		return err
+	}
+	defer localstackC.Terminate(ctx)
+
+	if err := os.Setenv("AWS_ENDPOINT_URL", addr); err != nil {
+		return err
+	}
+
+	dirs := []string{
```

Comment:
What if we take this outside of the function?

Reply:
I don't think this list will be used outside of this function